docusaurus/docs-internals/editor-controls/data-controls/inputText.md

```diff
@@ -0,0 +1,190 @@
+| `position`        | `number`                                                                                                                                                                        |    -    | The position of the control in toolbar                                                                                                                                                                                            |
+| `devices`         | `"all"`/`"desktop"`/`"responsive"`                                                                                                                                              | `"all"` | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices. |
+| `disabled`        | `boolean`                                                                                                                                                                       |  false  | Configure the condition under which the control is disabled or enabled.                                                                                                                                                           |
+| `config.size`     | `"small"` / `"medium"` / `"large"` / `"xlarge"` `"auto"`                                                                                                                        |    -    | The return value of the control when it is enabled                                                                                                                                                                                |
+| `helper.content`  | `string`                                                                                                                                                                        |    -    | If provided, displays an icon next to the label. When hovering over this icon, a tooltip with additional information appears                                                                                                      |
+| `helper.enabled`  | `boolean`                                                                                                                                                                       |    -    | Configure the condition under which the helper is disabled or enabled.                                                                                                                                                            |
```

Comment:
`enabled` is missing in helper props

![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/d0cfa5fe-5c20-48d9-a462-7cc9c0cd370d)